recipe/meta.yaml

```diff
@@ -18,8 +18,8 @@ requirements:
     - python
     - 'configurable-http-proxy >=4.1'
     - cylc-flow =={{ version }}
-    - cylc-uiserver ==0.3.0
-    - cylc-ui ==0.3.0
+    - cylc-uiserver ==0.4.0
+    - cylc-ui ==0.4.0
```

Comment:
Not critical but this dependency is not necessary:

```suggestion
```

Reply:
I know, but it doesn’t really matter, and as we discussed this metapackage is dead-ending soon.

Reply:
But OK, I’ll commit the change, it’ll be less confusing for casual observers!